tests/integration/evidence/keeper/infraction_test.go

```diff
@@ -39,7 +39,7 @@ import (
 	banktypes "github.com/cosmos/cosmos-sdk/x/bank/types"
 	consensusparamtypes "github.com/cosmos/cosmos-sdk/x/consensus/types"
 	minttypes "github.com/cosmos/cosmos-sdk/x/mint/types"
-	paramtypes "github.com/cosmos/cosmos-sdk/x/params/types"
```

Comment:
i think we should be able to drop this dep?

Reply:
It's under tests/integration, so it's fine. We still test params for x/tx amino support